fury/ui/tests/test_helpers.py

```diff
@@ -1,9 +1,11 @@
 """Test helpers fonction ."""
 
+from fury.ui.containers import GridUI
+from fury.actor import Container
```

Comment:
Your circular is due to this which makes complete sense.

I recommend you create custom objects for your tests and not use `GridUI` or `Container`.

look [here](https://github.com/dipy/dipy/blob/master/dipy/workflows/tests/workflow_tests_utils.py) for example and their [dummy tests](https://github.com/dipy/dipy/blob/master/dipy/workflows/tests/test_iap.py#L79).

So, create custom class, inheriting from `ui.core.UI`

Reply:
Hey @skoudoro, besides this there is another circular import [here](https://github.com/fury-gl/fury/blob/8de602803381c33072775744db53c1329020d046/fury/layout.py#L5). Following the custom class track, I think a `DummyUI` class that inherits from `UI` should be created outside the `ui` sub-module, then the `is_ui` method will not be needed anymore and there will be no circular imports. Is this what you were implying?

Reply:
Please, see my PR and feel free to merge it.